README.md

```diff
@@ -8,56 +8,108 @@
 [![Documentation badge](https://img.shields.io/readthedocs/fiware-iotagent-ul.svg)](http://fiware-iotagent-ul.readthedocs.org/en/latest/?badge=latest)
```

Comment:
Link to docker/README.md should be added. This comes from here: https://github.com/telefonicaid/iotagent-ul/pull/310#discussion_r235874606

Reply:
Fixed - 5661d11